### PR TITLE
[HIPIFY][doc] Readme and Changelog updates

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,208 +1,295 @@
...
+* New options:
+  * `--default-preprocessor` (synonymous with `--skip-excluded-preprocessor-conditional-blocks`)
+  * `--no-undocumented-features`
+  * `--no-warnings-on-undocumented-features`
+  * `--versions`
+
+### Fixes
+
+* Accessing `half2 struct` members (undocumented feature)
+* Moved `INSTALL` to the `bin` subfolder
```
Comment (emankov) [inline CHANGELOG.md]: INSTALL is a cmake operation, so it is better to reword to `Retarget INSTALL to bin subfolder`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,208 +1,295 @@
-# Change Log for HIPIFY
+# Changelog for HIPIFY
 
-Full documentation for HIPIFY is available at [hipify.readthedocs.io](https://hipify.readthedocs.io/en/latest/).
+Documentation for HIPIFY is available at
+[[HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/)](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
```
Comment (emankov) [inline CHANGELOG.md]: [https://rocmdocs.amd.com/projects/HIPIFY/en/latest/](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,208 +1,295 @@
...
+* CUDA 12.2.0 support
+* cuDNN 8.9.2 support
+* LLVM 16.0.6 support
+* Initial rocSPARSE support
+* Initial `CUDA2ROC` documentation generation for rocBLAS, rocSPARSE, and MIOpen:
+  * In separate files: `hipify-clang --md --doc-format=full --doc-roc=separate`
+  * In one file: `hipify-clang --md --doc-format=full --doc-roc=joint`
+* New options:
+  * `--use-hip-data-types` (Use 'hipDataType' instead of 'hipblasDatatype_t' or 'rocblas_datatype')
+  * `--doc-roc=\<value\>` (ROC documentation generation: `skip` (default), `separate`, and `joint`; the
```
Comment (emankov) [inline CHANGELOG.md]: Probably, an excessive line break

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,208 +1,295 @@
...
+
+### Additions
+
+* CUDA 11.5.1 support
+* cuDNN 8.3.2 support
+
+### Fixes
+
+* Hipification of `cuOccupancyMaxPotentialBlockSize` and
+  `cuOccupancyMaxPotentialBlockSizeWithFlags`
```
Comment (emankov) [inline CHANGELOG.md:137]: Probably, an excessive line break

File: README.md [hunk 5]
```diff
@@ -1,66 +1,55 @@
...
 
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
-### <a name="how-to-build-CUDA2HIP-documentation"></a>How to build CUDA2HIP documentation
-
-Run the following `hipify-clang` command to build the documentation for CUDA APIs supported by HIP (CUDA2HIP) locally.
+To build `CUDA2HIP` (CUDA APIs supported by HIP) documentation, run the following `hipify-clang`
+command. This builds the same content as
+[Supported CUDA APIs](./docs/supported_apis.md#supported-cuda-apis).
```
Comment (emankov) [inline README.md:44]: Actually no, to build the same CUDA2HIP a few more command lines are needed:
hipify-clang --md --doc-format=full  --doc-roc=joint
hipify-clang --md --doc-format=full  --doc-roc=separate

File: README.md [hunk 6]
```diff
@@ -1,66 +1,55 @@
...
 
-## <a name="documentation"></a>Documentation
+## Documentation
 
-Information about HIPIFY and other user topics is found in the [HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
+Documentation for HIPIFY is available at
+[[HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/)](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
 
-### <a name="how-to-build-HIPIFY-documentation"></a>How to build HIPIFY documentation
+To build our documentation locally, run the following code.
```
Comment (emankov) [inline README.md:32]: You breaks the TOC here

File: README.md [hunk 7]
```diff
@@ -1,66 +1,55 @@
...
 
 pip3 install -r .sphinx/requirements.txt
 
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
-### <a name="how-to-build-CUDA2HIP-documentation"></a>How to build CUDA2HIP documentation
-
-Run the following `hipify-clang` command to build the documentation for CUDA APIs supported by HIP (CUDA2HIP) locally.
+To build `CUDA2HIP` (CUDA APIs supported by HIP) documentation, run the following `hipify-clang`
```
Comment (emankov) [inline README.md:42]: You breaks the TOC here

File: README.md [hunk 8]
```diff
@@ -1,66 +1,55 @@
...
-
-AMD, the AMD Arrow logo, and combinations thereof are trademarks of Advanced Micro Devices, Inc. Other product names used in this publication are for identification purposes only and may be trademarks of their respective companies.
+To generate this documentation in CSV, use the `--csv` option instead of `--md`. Instead of using
+the `full` format, you can also build in `strict` or `compact` format.
 
-Copyright &copy; 2016-2023 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (emankov) [inline README.md:66]: Why Disclaimer and Copyright Notice are removed?

**Review by emankov — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (LisaDelaney) [reply CHANGELOG.md:137]: I'm keeping the lines at 80 characters or less.

**Review by emankov — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**